x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -96,60 +95,63 @@ export const getColumns = (
       field: 'method',
       name: i18n.COLUMN_METHOD,
       truncateText: true,
+      width: '16%',
     },
     {
       field: 'severity',
       name: i18n.COLUMN_SEVERITY,
       render: (value: TableData['severity']) => <SeverityBadge value={value} />,
       truncateText: true,
+      width: '16%',
     },
     {
-      field: 'lastCompletedRun',
+      field: 'statusDate',
       name: i18n.COLUMN_LAST_COMPLETE_RUN,
-      render: (value: TableData['lastCompletedRun']) => {
+      render: (value: TableData['statusDate']) => {
         return value == null ? (
           getEmptyTagValue()
         ) : (
-          <PreferenceFormattedDate value={new Date(value)} />
+          <FormattedDate value={value} fieldName={i18n.COLUMN_LAST_COMPLETE_RUN} />
         );
       },
       sortable: true,
       truncateText: true,
-      width: '16%',
+      width: '20%',
     },
     {
-      field: 'lastResponse',
+      field: 'status',
       name: i18n.COLUMN_LAST_RESPONSE,
-      render: (value: TableData['lastResponse']) => {
-        return value == null ? (
-          getEmptyTagValue()
-        ) : (
+      render: (value: TableData['status']) => {
+        const color =
+          value == null
+            ? 'subdued'
+            : value === 'succeeded'
+            ? 'success'
+            : value === 'failed'
+            ? 'danger'
+            : value === 'executing'
+            ? 'warning'
+            : 'subdued';
```

Comment:
I think @dhurley14 has one more value which is 'going to run'?

```ts
export type RuleStatusString = 'succeeded' | 'failed' | 'going to run' | 'executing';
```

But I don't see it being set on the backend. We might need to delete `going to run` from the backend if that is the case to avoid confusion.

Reply:
I did this with @dhurley14 , so it should be good

Reply:
and `going to run` is not there